src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AWS.Distro.OpenTelemetry.AutoInstrumentation.csproj

```diff
@@ -20,6 +20,9 @@
     <PackageReference Include="OpenTelemetry.Extensions.Propagators" Version="1.9.0" />
     <PackageReference Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
     <PackageReference Include="OpenTelemetry.ResourceDetectors.AWS" Version="1.4.0-beta.1" />
+    <!-- temp use private otel repo as local dependency -->
+    <!-- <PackageReference Include="OpenTelemetry.Instrumentation.AWS" Version="1.1.0-beta.4" /> -->
+    <ProjectReference Include="../../../opentelemetry-dotnet-contrib/src/OpenTelemetry.Instrumentation.AWS/OpenTelemetry.Instrumentation.AWS.csproj" />
```

Comment:
Why do you add this? Will this work in Github workflow?

Reply:
No longer needed now that upstream commits have been copied to local fork of OpenTelemetry repo.